msteams-platform/tabs/how-to/authentication/tab-sso-overview.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Overview to authentication for tabs using SSO in Teams with Azure AD
+description: Overview to SSO authentication in Teams and how to use it in tabs
+ms.topic: conceptual
+ms.localizationpriority: medium
+keywords: teams authentication tabs Microsoft Azure Active Directory (Azure AD) SSO access token app manifest 
+---
+# Enable single sign-on in a tab app
+
+Single sign-on (SSO) allows a user to access an application or a web service after signing-in only once. The app users never have to go through authentication again.
+
+With SSO in Teams, app users have an added advantage. After logging into Teams using Microsoft or Microsoft 365 account, app users don't need to sign in again to use your app. App users can access your app within Teams even on any other device with access granted through Azure AD.
+
+Here's what you'll learn in this section:
+
+1. **SSO user experience**: Teams offers your app users a true SSO experience. App users can use your app without signing in again.
+2. **SSO in Teams at runtime**: Your tab app interacts with Azure AD at runtime for one-time authenticating and authorizing of your app users.
+3. **Enable SSO for your tab app**: Implement the tasks involved to implement SSO in your tab app.
+
+## SSO user experience in Teams
+
+App users sign in to Teams using either personal Microsoft account or Microsoft 365 account. You can take advantage of this, and use SSO to authenticate and authorize the app users.
+
+&nbsp;&nbsp;&nbsp;&nbsp; :::image type="content" source="../../../assets/images/authentication/teams-sso-tabs/teams-sso-ux.png" alt-text="SSO user experience in a Teams tab app" border="false":::
+
+- Teams authenticates and stores the identity of its app user.
+- Your tab app uses the stored identity of the app user who is already validated by Teams.
+- The app user needs to give consent to Teams for using the identity to access for using your tab app.
```

Comment:
The app user or a tenant administrator can give consent. The administrator can consent for all users, removing the need to prompt the user.

Reply:
Some permissions require an admin to consent in fact - users can't grant permission to very powerful permission scopes such as Sites.ReadWrite.All (allows reading and writing all SharePoint and OneDrive assets in the whole tenant).